marimo/_server/api/endpoints/ws.py

```diff
@@ -95,6 +106,109 @@ async def websocket_endpoint(
     ).start()
 
 
+@dataclass
+class Cell:
```

Comment:
Bit of a Namespace conflict with other "Cell" classes. Thoughts on EditingCell or something?